pkg/apis/autoscaling/v1/defaults.go

```diff
@@ -33,4 +33,10 @@ func SetDefaults_HorizontalPodAutoscaler(obj *autoscalingv1.HorizontalPodAutosca
 
 	// NB: we apply a default for CPU utilization in conversion because
 	// we need access to the annotations to properly apply the default.
+	// Set autoscalingv1.HorizontalPodAutoscalerSpec.TargetCPUUtilizationPercentage
+	// to 100, which has the same meaning as unset.
```

Comment:
That's not entirely true.  Setting CPU causes CPU to be considered as a metric in calculations.  If CPU usage is very high, this could lead to a different scale-up than you intended.